### DISCUSSION TITLE
feature - Podcast manager

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#781] [Ideas]
Not in the sense of something like GPodder, but ability to automatically save state of time when pausing/stopping a podcast. So that you can later pick up where you left off. Saw some mpd daemon client that did this, but it's python and hasn't been worked in a long time. So the libraries it uses are no longer maintained.

**Comment by bonelifer** [comment]
Ok, just found this and it compiles still. would make a nice added feature for podcasts. Unless I'm missing something and this is already supported.

https://github.com/Mic92/mpdstated

**Comment by jcorporation** [comment]
I also thought about implementing something like this.

Podcasts could be managed e. g. with podget.

- simple script to run podget and update the mpd database
- a myMPD saved search to show new podcast episodes
- Save state of played time of an episode

**Comment by bonelifer** [comment]
My biggest reason for wanting to use GPodder is some of the podcasts like NBC Nightly news don't bother tagging their files. GPodder grabs the metadata from the RSS feed and applies it, and adds the Podcast genre. It does have a command line client, ```gpo```. I used to run the gpo command from cron all the time.

**Comment by jcorporation** [comment]
Insufficient metadata is often a problem. That’s why I do not want to write a full podcast client, but some features to work better with podcasts are possible.

**Comment by bonelifer** [comment]
That does sound good.  Basically mympd would have a config page for podcast where the user would be able to give to mympd a script to call to do the heavy lifting. Seems like the best option. In my instance it would contain:
```
gpo update
gpo download
mpc update
```
I assume that the podcast folder would best just be a folder in their music collection to make things simpler.  Thanks for considering this.

When I get my mpd setup working again(managed to hose it beyond repairability), I could look into various github podcatchers and the commands each one need to update and submit shell scripts for them. Maybe a separate mympd-podcast-updater repo for users to submit their own shell scripts for any obscure, personally written podcatchers. This would help new users decide for themselves.

**Comment by jcorporation** [comment]
Calling a script is already possible through the lua script integration. With v9.4.0 users can import scripts from my GitHub repo.

**Comment by jcorporation** [comment]
#844 

Adding a [mpdstated](https://github.com/Mic92/mpdstated) like feature could be the first step.

**Comment by bonelifer** [comment]
Still interested in this. Just haven't got around to setting up my mpd setup due to weird power issues with the server, So I'm going to just setup the hp thinclient for mpd and misc functions,

**Reply by jcorporation** [reply]
Since myMPD v10.1.0 there is an elapsed sticker, but no function uses it at the moment. A resume playback function could be possible.
